processing/src/main/java/org/apache/druid/math/expr/ExprEval.java

```diff
@@ -743,25 +749,37 @@ private NumericExprEval(@Nullable Number value)
     @Override
     public final int asInt()
     {
+      if (value == null) {
+        assert NullHandling.replaceWithDefault();
+        return 0;
+      }
       return value.intValue();
     }
 
     @Override
     public final long asLong()
     {
+      if (value == null) {
+        assert NullHandling.replaceWithDefault();
+        return 0L;
+      }
       return value.longValue();
     }
 
     @Override
     public final double asDouble()
     {
+      if (value == null) {
+        assert NullHandling.replaceWithDefault();
+        return 0.0;
+      }
       return value.doubleValue();
     }
 
     @Override
     public boolean isNumericNull()
```

Comment:
I know this is a comment on unchanged code, but wondering how you feel about renaming this to just `isNull`?  I've never quite understand why the `Numeric` part was important.